packages/auth/src/accessToken/routes.ts

```diff
@@ -119,20 +120,25 @@ async function rotateToken(
     newToken = await deps.accessTokenService.rotate(ctx.accessToken.id, trx)
 
     if (!newToken) {
-      ctx.throw()
+      throw new Error('invalid access token')
```

Comment:
Why not throw an internal server error with this error message? I know that this is kind of what we are doing here, but should we do a `ctx.throw`?

Reply:
I think because we end up catching the error and then call `ctx.throw` below